tools/profileBuilder/latest.go

```diff
@@ -78,11 +78,10 @@ func (latest LatestStrategy) Enumerate(cancel <-chan struct{}) collection.Enumer
 				return
 			}
 
-			version := pathMatches[2]
+			version := pathMatches[3]
 			currentGroup := operationGroup{
 				provider:     pathMatches[1],
-				resourceType: pathMatches[2],
-				group:        pathMatches[3],
+				resourceType: pathMatches[4],
```

Comment:
>resourceType: pathMatches[4], [](start = 4, length = 29)

does this also work with services that have a `parentResourcePath` ? What about graphrbac? Their paths are not like the rest of ARM

Reply:
Yep, I've been adding in support for broader and broader sets. The regexp itself is structured to take care of those differences at the moment.